### ISSUE TITLE
RuntimeError: Prohibited to create memoryview without Py_buffer

### ISSUE THREAD

**Issue by petaflot** [pdfarranger/pdfarranger#329]
this happens when trying to save files.

app-text/pdfarranger-1.6.2 on Gentoo 17.1 desktop with python 3.7.9

  File "/usr/lib/python3.7/site-packages/pdfarranger/pdfarranger.py", line 748, in choose_export_pdf_name
    self.save(only_selected, file_out)
  File "/usr/lib/python3.7/site-packages/pdfarranger/pdfarranger.py", line 210, in wrapper
    func(*args, **kwargs)
  File "/usr/lib/python3.7/site-packages/pdfarranger/pdfarranger.py", line 795, in save
    m = metadata.merge(self.metadata, self.pdfqueue)
  File "/usr/lib/python3.7/site-packages/pdfarranger/metadata.py", line 74, in merge
    doc = pikepdf.open(p.copyname)
  File "/usr/lib/python3.7/site-packages/pikepdf/__init__.py", line 72, in open
    return Pdf.open(*args, **kwargs)
  File "/usr/lib/python3.7/site-packages/pikepdf/_methods.py", line 837, in open
    access_mode=access_mode,
RuntimeError: Prohibited to create memoryview without Py_buffer

**Comment by angsch** [comment]
What version of pikepdf do you use (`pip show pikepdf`)?

This bug may be a duplicate of https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=975202

**Comment by jeromerobert** [comment]
It looks like version 1.18.0. Running `python3 -c 'import pikepdf'` should confirm that this issue is not related to pdfarranger. It's more likely somewhere between python, pikepdf and pypind11 and should be reported to the Gentoo pikepdf packager.
`pip install pikepdf` or `pip install pikepdf==1.19.3` is probably a workaround.

**Comment by jeromerobert** [comment]
I confirm that pikepdf 1.8.0 is not compatible with pybind11 2.6.0. You need https://github.com/pikepdf/pikepdf/commit/5394b7855eb87f637316432b2b597294adcb974e which is in pikepdf 1.19.0.

**Comment by petaflot** [comment]
meta@gentx ~ $ pip show pikepdf
Name: pikepdf
Version: 1.18.0
Summary: Read and write PDFs with Python, powered by qpdf
Home-page: https://github.com/pikepdf/pikepdf
Author: James R. Barlow
Author-email: james@purplerock.ca
License: UNKNOWN
Location: /usr/lib/python3.7/site-packages
Requires: lxml, Pillow
Required-by: pdfarranger

**Comment by petaflot** [comment]
> pip install pikepdf==1.19.3

this works :-)


thanks jeromerobert
